codegen/stub_api.py

```diff
@@ -749,7 +759,20 @@ def process(self, dirname: str, filename: str) -> None:
         s += "from ansys.pyensight.core.ensobj import ENSOBJ\n"
         s += "from ansys.pyensight.core import ensobjlist\n"
         s += "ENSIMPORTS"
-        s += "from typing import Any, List, Type, Union, Optional, Tuple\n"
+        s += "from typing import Any, List, Type, Union, Optional, Tuple, TYPE_CHECKING\n"
```

Comment:
I am there creating a subclass of SimpleNamespace where I am adding the attributes for the utils. Note that I am not creating any instance, just declaring it